tests/bench.py

```
@@ -0,0 +1,60 @@
+import pytest
+import os
+import platform
+import subprocess
+import pytest
```

Ruff (0.8.2)

Redefinition of unused `pytest` from line 1

Remove definition: `pytest`

(F811)